Basic-Car-Maintenance/Shared/Settings/ViewModels/SettingsViewModel.swift

```diff
@@ -20,6 +22,41 @@ final class SettingsViewModel: ObservableObject {
         self.authenticationViewModel = authenticationViewModel
     }
     
+    let urls: [String: URL] = [
+        "mikaelacaronProfile": URL(string: "https://github.com/mikaelacaron")!,
+        "Basic-Car-MaintenanceRepo": URL(string: "https://github.com/mikaelacaron/Basic-Car-Maintenance")!,
+        "bugReport": URL(string: "https://github.com/mikaelacaron/Basic-Car-Maintenance/issues")!
+    ]
+    
+    let staticContributors: [String] = [
+        "mikaelacaron",
+        "shloak17107",
+        "prabal4546",
+        "NiklasOemler",
+        "vcapilladeveloper"
```

Comment:
This is directly copied from brain marks, please customize this for this project. It doesn't make sense to have a list of names from people who have not contributed to this project, and the staticContributors doesn't really make sense to have at all (as a predefined list, if anything you could just include my username and that's it)